public/blog-posts/liveness-2-and-beyond.md

```diff
@@ -0,0 +1,318 @@
+# Liveness 2 and beyond
```

Comment:
Blog posts are not looking good on mobile

![image](https://github.com/defi-wonderland/web/assets/84932007/d8e8bace-a900-48f5-9c8e-4d78b70c11bb)